azurelinuxagent/common/conf.py

```diff
@@ -304,6 +295,14 @@ def get_password_cryptid(conf=__conf__):
     return conf.get("Provisioning.PasswordCryptId", "6")
 
 
+def get_provisioning_agent(conf=__conf__):
+    return conf.get("Provisioning.Agent", "auto")
+
+
+def get_provision_enabled(conf=__conf__):
```

Comment:
I'd add a short comment explaining that the 'enabled' setting got replace with 'agent'